xwiki-platform-distribution/xwiki-platform-distribution-war-legacydependencies/pom.xml

```diff
@@ -228,6 +232,37 @@
          </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-legacy-rendering-wikimacro-api</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <exclusion>
+          <artifactId>xwiki-platform-rendering-wikimacro-api</artifactId>
```

Comment:
This one is not a transitive dependency so no need to exclude it.

Reply:
After verification this exclusion has no effect and as the dependency is not drawn from `xwiki-platform-legacy-rendering-wikimacro-api`, removing.